installing/installing_helm/install-helm-customization.adoc

```diff
@@ -51,16 +50,78 @@ $ helm install -n stackrox --create-namespace \
 ----
 <1> Use the `-f` option to specify the paths for your YAML configuration files.
 
+[id=generate-inti-bundle]
+== Generating an init bundle
+
+To create a secured cluster, you must create an init bundle. The secured cluster uses this bundle to authenticate with Central.
+
+You can create an init bundle using the the roxctl CLI or from the RHACS portal.
+
+[.discreet]
+=== Generating an init bundle by using roxctl CLI
+
 [id="install-roxctl-cli"]
-== Installing the roxctl CLI
+==== Installing the roxctl CLI
```

Comment:
To generate an init bundle, one would need the `roxctl` CLI. Since this is the installation document I've included the steps to download and install roxctl CLI. 
- Should we just remove this and add a link to downloading the roxctl CLI instead ?

Reply:
So I gave it another thought and removed the installing CLI section and added it as an additional resource.